pkgs/desktops/gnome-3/core/gnome-tour/default.nix

```diff
@@ -15,9 +16,11 @@
 , gnome3
 , libhandy
 , librsvg
+, rustc
+, cargo
```

Comment:
Nitpick: why the separate parameters instead of taking them from `rustPlatform`?

Reply:
cargo doesn't come from rustPlatform, so I thought "they're inherited for a reason".